server/odc-service/src/main/java/com/oceanbase/odc/service/flow/util/FlowTaskUtil.java

```diff
@@ -256,10 +257,26 @@ public static String getSchemaName(@NonNull DelegateExecution execution) {
         return internalGet(value, String.class).orElseThrow(() -> new VerifyException("Schema name is absent"));
     }
 
+    // use setConnectionConfigList instead
+    @Deprecated
```

Comment:
why deprecate it ?

there exists tasks already use connectionConfig,
connectionConfigList should only referenced by multiple database change.

Reply:
`setConnectionConfigList` can cover all situations of `setConnectionConfig` and we prefer using the new one~

Reply:
list is not clear for all tasks except mulitple database change.
the commit accepted, anyway i resource my opnion.